vj4/handler/base.py

```diff
@@ -51,6 +51,11 @@ class HandlerBase(setting.SettingMixin):
           domain.get(self.domain_id),
           domain.get_user(self.domain_id, uid),
           blacklist.get(self.remote_ip))
+      sentry = app.Application().sentry_middleware
```

Comment:
this will raise `AttributeError` when `sentry_middleware` is not assigned.

`app.Application()` is relatively expensive, consider assigning at class level (i.e. put `app = app.Application()` at line 36)

Reply:
ok I see the `sentry_middleware = None`, reverting point 1